cmd/copy.go

```diff
@@ -650,10 +650,10 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 		}
 		// Disabling blob tier override, when copying block -> block blob or page -> page blob, blob tier will be kept,
 		// For s3 and file, only hot block blob tier is supported.
-		if cooked.blockBlobTier != common.EBlockBlobTier.None() ||
-			cooked.pageBlobTier != common.EPageBlobTier.None() {
-			return cooked, fmt.Errorf("blob-tier is not supported while copying from service to service")
-		}
```

Comment:
is this meant to be commented out?

Reply:
S2S blob tier copy is now supported.

Reply:
FromTo scenarios where I think tier can be set using block-blob-tier or page-blob-tier flags are LocalBlob(), BlobBlob(), FileBlob(), & S3Blob(). Is that correct? If yes, I'll add the below if statement to check the error.

```
if (cooked.blockBlobTier != common.EBlockBlobTier.None() || cooked.pageBlobTier != common.EPageBlobTier.None()) &&
   cooked.fromTo.To() != common.ELocation.Blob() {
   return cooked, fmt.Errorf("blob-tier is not supported while copying from service to service")
}
```